docs/Set-PSReadLineOption.md

```diff
@@ -136,11 +136,14 @@ For example, if my prompt function looked like:
 
 Then set:
 
-    Set-PSReadLineOption -PromptText "# "
+    Set-PSReadLineOption -PromptText "# ", "!"
```

Comment:
Maybe use `"! "` (add a space) so they are the same length.

I'm guessing if they aren't the same length, rendering will get messed up.

Reply:
You're right. I meant to, but obviously didn't. :)